wweval/R/ms_fig4.R

```diff
@@ -1,53 +1,81 @@
+#' Make a summary of the with and without wastewater comparison scores
+#'
+#' @param scores a tibble of scores for each model, horizon day, forecast date
+#' and location for the subset of forecasts used in the head-to-head comparison
+#'
+#' @return a list of two tables with summary scores, one overall and one by
+#' forecast vs nowcast
+#' @export
+make_fig4_results_table <- function(scores) {
+  # Overall avg crps, bias, absolute error etc
+  scores_overall <- scores |>
+    dplyr::group_by(model) |>
+    dplyr::summarise(
+      avg_crps = mean(crps),
+      avg_bias = mean(bias),
+      avg_ae = mean(ae_median)
+    )
+
+  # Above was averaged across models, get avg of rel_crps
+  overall_all_time_rel_crps <- scores |>
+    compute_relative_crps(id_cols = c(
+      "location",
+      "forecast_date", "date", "horizon"
+    )) |>
+    dplyr::summarize(mean_rel_crps = mean(rel_crps, na.rm = TRUE))
```

Comment:
Same comment as above.

Reply:
Same thing, I have both being computed but reporting only relative of means

Reply:
Will remove

Reply:
Now fixed